svid/x509svid/tests/check_source.c

```diff
@@ -0,0 +1,47 @@
+#include "svid/x509svid/src/source.h"
+#include <check.h>
+
+START_TEST(test_x509svid_Source_GetX509SVID) {}
+END_TEST
+
+START_TEST(test_x509svid_SourceFromSVID) {}
+END_TEST
+
+START_TEST(test_x509svid_SourceFromSource)
```

Comment:
Add some comment for each test (preconditions, function under test and postconditions for example)